psutil/_psutil_windows.c

```diff
@@ -18,9 +18,14 @@
 #include <tchar.h>
 #include <tlhelp32.h>
 #include <winsock2.h>
+#if (_WIN32_WINNT >= 0x0600) // Windows Vista, 7, 8, 8.1, 10
+#include <ws2tcpip.h>
+#endif
 #include <iphlpapi.h>
 #include <wtsapi32.h>
+#if (_WIN32_WINNT < 0x0600) // Windows XP / 2000
 #include <ws2tcpip.h>
+#endif
```

Comment:
This actually doesn't need to be included. The issue was that it matters where it's located in the list of includes, but after further testing, this header is not required for Win XP.